examples/typescript-server.ts

```diff
@@ -48,7 +48,11 @@ server.use(cors())
 server.get('/', opts, getHelloHandler)
 server.get('/stream', getStreamHandler)
 
-server.listen(3000, err => {
-  if (err) throw err
-  console.log(`server listening on ${server.server.address().port}`)
```

Comment:
I think we should keep the previous callback thingy as well. This code is used only to validate the types and it does not run.